content/topics/Analyze/causal-inference/introduction/synthetic-control-methods-intro.md

```diff
@@ -0,0 +1,268 @@
+- Similarly, $\mathbf{X}_0$ identifies pre-intervention characteristics for the untreated units.
+
+Such variables in our example can be identified as income and retail price of cigarettes, just to mention a few. 
+
+Let $Y_{jt}$ be the value of the outcome for unit $j$ at time $t$.
+
+For a post-intervention period $t$ (with $t \geq T_0$) the treatment effect with the synthetic control estimator is:
+
+$
+\hat\tau_{1t} = Y_{1t} - \sum_{j=2}^{J+1} w_j^{'} Y_{jt}
+$
+
+Now that we have set a basis for the theoretical aspect of synthetic control methods, let's dive into the implementation in R. 
+
+## Implementation in R 
+
+The implementation steps follow [this example](https://github.com/edunford/tidysynth/blob/master/README.md). 
+
+The package we are going to be using is called *tidysynth*, with functionalities specific for the implementation of this method. 
+
+Let's start with installing and loading the package. 
+
+{{% codeblock %}}
+```R
+install.packages('tidysynth')
+library(tidysynth)
+```
+{{% /codeblock %}}
+
+The dataset we are going to use in this tutorial is the *smoking* dataset, based on the paper by [Abadie et al. (2010)](https://www.tandfonline.com/doi/abs/10.1198/jasa.2009.ap08746), which evaluates the impact of tobacco control program on cigarette consumption in California.
+
+{{% codeblock %}}
+```R
+require(tidysynth)
+data("smoking")
+View(smoking)
+```
+{{% /codeblock %}}
+
+<p align = "center">
+<img src = "../images/3_synthetic.png" width="700">
+</p>
+
+
+The method aims to generate a synthetic California using information from a subset of control states (the “donor pool”) where a similar law was not implemented. The donor pool is the subset of case comparisons from which information is borrowed to generate a synthetic version of the treated unit (“California”). 
+
+First, we initiate the synthetic control object:
+
+{{% codeblock %}}
+```R
+smoking_out <-
+  
+  smoking %>%
+  synthetic_control(outcome = cigsale, 
+                    unit = state,
+                    time = year,
+                    i_unit = "California",
+                    i_time = 1988,
+                    generate_placebos=T)
+``` 
+{{% /codeblock %}}
+
+In this step, we create the synthetic control object using the synthetic_control function. We specify the outcome variable (cigsale), the unit index (state), and the time index (year). We also identify the intervention unit (California) and the intervention time (1988). The *generate_placebos* option is set to TRUE to allow for placebo controls.
+
+Next, we generate predictors for the model. Predictors are the variables used to characterize each unit in the dataset before the intervention, they are the features or attributes based on which the similarity between the treated unit and control units is assessed. The synthetic control method uses these predictors to ensure that the synthetic control unit closely matches the treated unit in the pre-intervention period.
```

Comment:
would help to have subheadings for each step? name it as :

- Step 1: Initiate synthetic control object
- Step 2: Generate predictors 

and so on..